src/store/index.js

```diff
@@ -1,15 +1,7 @@
 import Vue from 'vue'
-import Vuex from 'vuex'
 
-Vue.use(Vuex)
-
-export default new Vuex.Store({
-  state: {
-  },
-  mutations: {
-  },
-  actions: {
-  },
-  modules: {
-  }
+const state = Vue.observable({
```

Comment:
That's some original approach here! Beware though that `Vue.observable`, being quite new (contrary to Vuex) has [worse tooling support](https://forum.vuejs.org/t/vue-observable-vs-vuex/56412).